tests/iface_loopback_action/iface_loopback_action_helper.py

```diff
@@ -464,8 +499,7 @@ def add_ptf_bond(ptfhost, port, bond_id, ip_addr):
     """
     try:
         bond_port = 'bond{}'.format(bond_id)
-        ptfhost.shell("ip link add {} type bond".format(bond_port))
-        ptfhost.shell("ip link set {} type bond miimon 100 mode 802.3ad".format(bond_port))
+        ptfhost.shell("teamd -t {} -d -c '{{\"runner\": {{\"name\": \"lacp\"}}}}'".format(bond_port))
```

Comment:
Did you see issue with ip link? Any reason to use teamd commands?

Reply:
Yes, on some of our testbed, the ip link bond is not working. But we haven't found the root cause yet. It is not related to the feature, looks like some environment issue.